test/e2e/kubernetes/workloads/statefulset.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: web
+spec:
+  selector:
+    matchLabels:
+      app: nginx
+  serviceName: "nginx"
+  podManagementPolicy: "Parallel"
+  replicas: 3 # by default is 1

```

Comment:
note to self get rid of the inline comment

Reply:
Is there a way in the aks-engine tests to scale up and down the replicas for this specific tests ?

Reply:
Should be able to add that to the tests. @jackfrancis what do you think?